src/env.rs

```diff
@@ -275,7 +281,7 @@ mod tests {
 
     #[test]
     fn test_find_environments() {
-        let config = toml! {
+        let config: PathBuf = toml! {
```

Comment:
This was actually a compile error for me on the tests, so I just fixed it. I don't know if it has gotten more strict with the newer versions of rust (I have 1.81.0)